src/workflows-app/RunDetails.test.js

```diff
@@ -580,4 +585,50 @@ describe('BaseRunDetails - render smoke test', () => {
     expect(targetRow.textContent).toContain(targetStartText);
     expect(targetRow.textContent).toContain(targetEndText);
   });
+
+  it('loads the call cache diff wizard', async () => {
+    const user = userEvent.setup();
+    await act(async () => render(h(BaseRunDetails, runDetailsProps)));
+    const showWizard = screen.getByTestId('cc-icon');
+    await user.click(showWizard); // Open the modal
```

Comment:
Just confirming that this modal can be opened in a screen-reader-accessible way. I think we're OK if the `Link` that says 'Call Cache Wizard' opens the modal. Otherwise we should add an aria-label to this icon and test it with `screen.getByLabelText`

Reply:
Oh, good call, we are not OK and I will make that change.